### PR TITLE
Environment CR webhook validation.

### PR DESCRIPTION
**Pull request by jparsai**
## Description:
This PR is to update webhook configs in managed-gitops that is implemented in application-api repo.

## JIRA Story:
https://issues.redhat.com/browse/GITOPSRVCE-573

### CODE REVIEW

**Comment by jparsai** [comment]
Thanks @Rizwana777, looks like you missed to submit review comment.

**Comment by Rizwana777** [comment]
> Thanks @Rizwana777, looks like you missed to submit review comment.

Yes done, Thanks @jparsai :)

File: tests-e2e/appstudio/webhook_test.go [hunk 1]
```diff
@@ -184,6 +184,37 @@ var _ = Describe("Webhook E2E tests", func() {
...
+			k8sClient, err = fixture.GetE2ETestUserWorkspaceKubeClient()
+			Expect(err).To(Succeed())
+
+			if !isWebhookInstalled("environments", k8sClient) {
+				Skip("skipping as environments webhook is not installed")
+			}
+
+			Expect(fixture.EnsureCleanSlate()).To(Succeed())
+
+			By("Validate that Environment name longer than 63 char is not allowed.")
```
[the remark this answers is not in the document]
Comment (jparsai) [reply tests-e2e/appstudio/webhook_test.go:197]: Yes, this test case is already there, but the comment in code is wrong, I will correct the comment. 
https://github.com/redhat-appstudio/managed-gitops/blob/e07faf9ebd2dd48d6e038e7b9ffd6c73f3037dc5/tests-e2e/appstudio/webhook_test.go#L215

[the remark this answers is not in the document]
Comment (Rizwana777) [reply tests-e2e/appstudio/webhook_test.go:197]: Thanks Jayendra!

**Review by jgwest — APPROVED** [review]
LGTM too, thanks @jparsai! And thank @Rizwana777 for reviewing!

**Pull request merged**